src/neo/Oracle/OraclePolicyContract.cs

```diff
@@ -0,0 +1,230 @@
+        private StackItem SetTimeOutMilliSeconds(ApplicationEngine engine, Array args)
+        {
+            StoreView snapshot = engine.Snapshot;
+            UInt160 account = GetOracleMultiSigAddress(snapshot);
+            if (!InteropService.Runtime.CheckWitnessInternal(engine, account)) return false;
+            if (BitConverter.ToInt32(args[0].GetSpan()) <= 0) return false;
+            int timeOutMilliSeconds = BitConverter.ToInt32(args[0].GetSpan());
+            StorageItem storage = snapshot.Storages.GetAndChange(CreateStorageKey(Prefix_TimeOutMilliSeconds));
+            storage.Value = BitConverter.GetBytes(timeOutMilliSeconds);
+            return true;
+        }
+
+        /// <summary>
+        /// Get timeout
+        /// </summary>
+        /// <param name="engine">VM</param>
+        /// <returns>value</returns>
+        [ContractMethod(1_00000000, ContractParameterType.Integer)]
+        private StackItem GetTimeOutMilliSeconds(ApplicationEngine engine, Array args)
+        {
+            return new Integer(GetTimeOutMilliSeconds(engine.Snapshot));
+        }
+
+        /// <summary>
+        /// Get timeout
+        /// </summary>
+        /// <param name="snapshot">snapshot</param>
+        /// <returns>value</returns>
+        public int GetTimeOutMilliSeconds(StoreView snapshot)
+        {
+            return BitConverter.ToInt32(snapshot.Storages[CreateStorageKey(Prefix_TimeOutMilliSeconds)].Value, 0);
+        }
+
+        /// <summary>
+        /// Set PerRequestFee
+        /// </summary>
+        /// <param name="engine">VM</param>
+        /// <param name="args">Parameter Array</param>
+        /// <returns>Returns true if the execution is successful, otherwise returns false</returns>
+        [ContractMethod(0_03000000, ContractParameterType.Boolean, ParameterTypes = new[] { ContractParameterType.Integer }, ParameterNames = new[] { "fee" })]
+        private StackItem SetPerRequestFee(ApplicationEngine engine, Array args)
+        {
+            StoreView snapshot = engine.Snapshot;
+            UInt160 account = GetOracleMultiSigAddress(snapshot);
+            if (!InteropService.Runtime.CheckWitnessInternal(engine, account)) return false;
+            if (BitConverter.ToInt32(args[0].GetSpan()) <= 0) return false;
+            int perRequestFee = BitConverter.ToInt32(args[0].GetSpan());
+            StorageItem storage = snapshot.Storages.GetAndChange(CreateStorageKey(Prefix_PerRequestFee));
+            storage.Value = BitConverter.GetBytes(perRequestFee);
+            return true;
+        }
+
+        /// <summary>
+        /// Get PerRequestFee
+        /// </summary>
+        /// <param name="engine">VM</param>
+        /// <param name="args">Parameter Array</param>
+        /// <returns>Value</returns>
+        [ContractMethod(1_00000000, ContractParameterType.Integer, SafeMethod = true)]
+        private StackItem GetPerRequestFee(ApplicationEngine engine, Array args)
```

Comment:
Remove args

Reply:
I have tried to remove it before, but this will cause an exception.This may be related to the loading mechanism of the native contract

Reply:
![image](https://user-images.githubusercontent.com/42929757/74699557-f2e71500-523b-11ea-90ad-abeb42e0a647.png)

Reply:
https://github.com/neo-project/neo/blob/331e4857cb4bbf738a90dd41157c288ade39bbd2/src/neo/SmartContract/Native/NativeContract.cs#L61

You are right, we always must use this pattern..